src/main/java/eu/ai4work/sws/exception/InValidFclFileException.java

```diff
@@ -0,0 +1,7 @@
+package eu.ai4work.sws.exception;
+
+public class InValidFclFileException extends RuntimeException {
```

Comment:
Minor general comment: `InValid` seems to be more difficult to read than `Invalid`, so I would suggest to make the `V` lowercase in all of the code.